### ISSUE TITLE
Make OpenSlides navigateable with keyboard and readable with screenreader

### ISSUE THREAD

**Issue by jsaalfeld** [OpenSlides/OpenSlides#3654]
Many people have issues with their eyesight and relay on screenreaders and navigation with the keyboard.

At the moment you can only rudimentary navigate OpenSlides with your keyboard. It would help if we have designated commands, to sufficiently use OpenSlides with your keyboard.

OpenSlides should be accessible with a screen reader, so that people can OpenSlides just be receiving the read out text by the computer, via a [refreshable braille display](https://en.wikipedia.org/wiki/Refreshable_braille_display) or other tools that enhance font sizes, change colors (etc.). This way OpenSlides would be much more accessible.

**Comment by rrenkert** [comment]
We are currently working hard on accessibility in the client and projector. See open issues there.
